sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 # Release History
 
-## 5.1.0-preview.1 (Unreleased)
+## 5.1.0-beta.1 (Unreleased)
 
 - We are now targeting the service's v3.1-preview.1 API as the default instead of v3.0.
 - We now have added support for opinion mining feature of Sentiment Analysis. To use this feature, you need to make sure you are using the service's v3.1-preview.1 API. To get this support pass `includeOpinionMining` as True when calling the `analyzeSentiment` endpoint.
+- The `length` and `offset` properties are added to the `SentenceSentiment` interface to represent the length of the sentence's text and its offset from the start of the document. The unit of distance used is UTF-16 code points.
```

Comment:
length and offset should also be added to `LinkedEntityMatch` and `CategorizedEntity`

Reply:
thanks, it is indeed missing a few ones. I added them all.